subprojects/base-services/src/main/java/org/gradle/internal/Factories.java

```diff
@@ -28,6 +30,20 @@ public T create() {
         };
     }
 
+    public static <T> Factory<T> toFactory(final Callable<T> callable) {
```

Comment:
Can we just change the API to use `Factory` vs `Callable`? Looking at the usages, it doesn't look like we have to have Callables

Reply:
Sure.  Done.